x-pack/platform/plugins/shared/lens/public/state_management/lens_slice.ts

```diff
@@ -136,7 +137,7 @@ export const getPreloadedState = ({
     datasourceStates,
     visualization: {
       state: null,
-      activeId: Object.keys(visualizationMap)[0] ?? null,
+      activeId: visualizationType ?? Object.keys(visualizationMap)[0] ?? null,
```

Comment:
This is a solution to the original bug mentioned in #218347. I don't this this really has that large of an affect as the `activeId` is correct once `isLoading` is `true`. But better safe than sorry.